easybib/recipes/role-phpapp.rb

```diff
@@ -16,6 +16,4 @@
 
 include_recipe 'tideways'
 
-if is_aws
```

Comment:
@baccenfutter yeah, to be able to test and configure it locally. I send PRs to all products (hopefully), to disable it by default though:
https://github.com/easybib/scholar-vagrant/pull/87
https://github.com/easybib/researchready/commit/3848dfcd15875128773d1a88ef5dad8ed4d0c9d9